Mixpanel/Mixpanel+HostWatchOS.m

```diff
@@ -14,7 +14,7 @@ @implementation Mixpanel (HostWatchOS)
 - (void)session:(WCSession *)session didReceiveMessage:(NSDictionary<NSString *, id> *)message {
     NSString *messageType = [Mixpanel messageTypeForWatchSessionMessage:message];
     if (messageType) {
-        if ([messageType isEqualToString:@"track"]) {
+        if ([messageType isEqualToString:@"track:properties:"]) {
```

Comment:
Peanut gallery says: you mean `NSStringFromSelector`, right?

@samgreen, ObjC style question: Is `NSSelectorFromString(@selector(...))` our preferred style in general? I don't see any instances of `NSStringFromSelector(@selector(...))` in mixpanel-iphone yet, but I do see (`git grep '@"[a-z][^ "]*:'`) some instances of e.g.

    NSSelectorFromString(@"imageForState:")
    NSSelectorFromString(@"didMoveToWindow")

that seem like they ought to be refactored into `@selector(...)`. Or is there a reason we don't do that in those cases?
I'm not proposing a specific competing style here; just trying to understand the rationale for our current style.

http://corporationunknown.com/blog/2014/03/24/to-nsstringfromselector-or-not/